jobs/kola-gcp.Jenkinsfile

```diff
@@ -44,27 +44,36 @@ if (s3_stream_dir == "") {
     s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
 }
 
-cosaPod(image: params.COREOS_ASSEMBLER_IMAGE,
-        memory: "256Mi", kvm: false,
-        secrets: ["aws-fcos-builds-bot-config", "gcp-kola-tests-config"]) {
+try { timeout(time: 120, unit: 'MINUTES') {
```

Comment:
Maybe 30 mins for GCP? Looking at past runs, it seems to take ~15 mins.

Reply:
done